static/app/views/settings/account/accountSecurity/index.tsx

```diff
@@ -139,72 +136,70 @@ class AccountSecurity extends AsyncView<Props> {
                   configureButton,
                   name,
                 } = auth;
+
                 return (
-                  <AuthenticatorPanelItem key={id}>
-                    <AuthenticatorHeader>
-                      <AuthenticatorTitle>
-                        <AuthenticatorStatus enabled={isEnrolled} />
-                        <AuthenticatorName>{name}</AuthenticatorName>
-                      </AuthenticatorTitle>
-
-                      <Actions>
-                        {!isBackupInterface && !isEnrolled && hasVerifiedEmail && (
+                  // eslint-disable-next-line react/jsx-key
```

Comment:
do we need to disable this?
Looks like the key is included?

Reply:
Yeah not sure why its in there and it looks like I added it :/